website/docs/components/application-state/partials/guidelines/guidelines.md

```diff
@@ -2,28 +2,114 @@
-- Provide a straightforward explanation of the problem or error.
-- Include suggestions or guidance for how the user can resolve the issue, if possible.
+**Center alignment**
+
+![Center aligned application state](/assets/components/application-state/application-state-alignment-center.png)
+
+**Left alignment**
+
+![Left aligned application state](/assets/components/application-state/application-state-alignment-left.png)
+
+## Media
+
+The media slot is used to add illustrations to increase the visual impact of the Application State.
+This provides additional visual prominence while also elevating the brand experience. If the illustration has a circular container, we recommend setting the `alignment` to `center`.
+
+![Empty state for vault secrets, guiding user to create new secrets or importing them](/assets/components/application-state/application-state-media-slot-spot-illustration-center-alignment.png)
+
+
+## Header
+
+### Icon
+
+When set, the icon is displayed side by side with the title.
+
+![Showing an icon left of the title, with some body text below it.](/assets/components/application-state/application-state-icon-usage.png)
+
+This is commonly used when displaying an error state for application failures. The icon must always be accompanied by a title.
+
+### Title
+
+The title should be short and provide a clear and concise message.
+
+### Error code
+
+If enabled and available, an error code will be shown, providing additional information associated with the title.
+
+## Body
+
+Focus on relevant information and avoid unnecessary details. If there is an error, include suggestions or guidance for how the user can resolve the issue, if possible. If no objects are found (zero/empty state), provide a brief explanation on how creating this new objective will benefit the user. 
+
+The body allows for two types of content: `text` and `generic`.
+
+![Showing two different kind of body content types, one as text and another as generic yielded content](/assets/components/application-state/application-state-body-content-types.png)
+
+
+## Footer
+The Application State supports up to three actions, including [Dropdown](/components/dropdown), [Standalone Link](/components/link/standalone) and [Button](/components/button). Use footer actions to redirect or guide users in solving errors/access issues with actionable steps.
+
+![A button set showing a dropdown, secondary button and stand alone link](/assets/components/application-state/application-state-footer-action-types.png)
+
+### Using buttons or links
+
+Buttons, along with links, are used to emphasize the importance of the action in the Application State with visual hierarchy, and are the most common actions used in the footer.
+
+!!! Dont
+
+When there is an empty state that occupies the majority of the page, do not display two exact actions at the same time in different areas of the UI. In this example, there is a primary button in the Page Header and in the Application State. 
```

Comment:
This phrasing seems awkward and unclear: "do not display two exact actions at the same time in different areas of the UI"

Reply:
Maybe try this?

```suggestion
When there is an empty state that occupies the majority of the page, do not display two similar actions in different areas of the UI. In this example, there is a primary button in the Page Header and in the Application State. 
```